_locales/en/messages.json

```diff
@@ -97,6 +97,15 @@
   "matches": {
     "message": "Matches:"
   },
+  "menu_description": {
```

Comment:
We don't anywhere else, and I don't want to get into the business of, trying to "categorize" the strings.

If the string is "Description", its name should be "description", and we should be able to use it anywhere we want to use that string.

Reply:
Okay!

Reply:
How will we differ labels and objects, i.e. for example ```Version:``` vs. ```version```?
I'd know the answer for ```Version $version$``` (that would be ```version_VERSION```)...